cub/cub/device/dispatch/dispatch_transform.cuh

```diff
@@ -346,16 +356,15 @@ struct dispatch_t<StableAddress,
   CUB_RUNTIME_FUNCTION _CCCL_FORCEINLINE cudaError_t Invoke(ActivePolicyT active_policy = {})
   {
     auto wrapped_policy = detail::transform::MakeTransformPolicyWrapper(active_policy);
+    auto seq            = ::cuda::std::index_sequence_for<RandomAccessIteratorsIn...>{};
 #ifdef _CUB_HAS_TRANSFORM_UBLKCP
     if constexpr (Algorithm::ublkcp == wrapped_policy.GetAlgorithm())
     {
-      return invoke_ublkcp_algorithm<ActivePolicyT>(::cuda::std::index_sequence_for<RandomAccessIteratorsIn...>{});
+      return invoke_ublkcp_algorithm<ActivePolicyT>(seq);
     }
     else
 #endif // _CUB_HAS_TRANSFORM_UBLKCP
-    {
-      return invoke_prefetch_algorithm(::cuda::std::index_sequence_for<RandomAccessIteratorsIn...>{}, wrapped_policy);
-    }
```

Comment:
Not a fan of the missing braces. 

I would be fine if we dropped the `else` because the `if` returns directly, but removing the braces is dangerous and against our coding guidelines

Reply:
Readded them.

> I would be fine if we dropped the `else` because the `if` returns directly

We can't I think, because we need to avoid instantiating both paths.

> removing the braces is dangerous

I don't see any danger with not having the braces that clang-format cannot detect.

>  and against our coding guidelines

Btw, I always wondered where I can read up on those. Can you point me at?